recipes/elasticsearch-async/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "elasticsearch-async" %}
+{% set version = "5.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "f085a566264e92e59afeab2a172a56a32b9408d5c0a8c19958b0590b52badfb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - aiohttp
+    - elasticsearch >=5.0.0
+
+test:
+  imports:
+    - elasticsearch_async
+
+about:
+  home: https://github.com/elastic/elasticsearch-py-async
+  license: Apache 2.0
+  license_family: Apache
```

Comment:
Is there a license file that we can include in the package?

Reply:
Wasn't part of the source tarball. Added it as part of the recipe now.

Reply:
Could you please raise an issue about this upstream?